hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -503,6 +507,28 @@ public AbfsInputStreamStatistics getStreamStatistics() {
     return streamStatistics;
   }
 
+  /**
+   * Since, the value of readAheadBytesRead counter could be arbitrary due to
```

Comment:
say what it is rather than why, e,g "number of bytes picked up in async readahead"